acceptance-tests/src/main/java/io/blueocean/ath/WaitUtil.java

```diff
@@ -127,6 +128,9 @@ public void click(By by) {
                     logger.info(String.format("Retry click successful on attempt " + i + " for %s", by.toString()));
                 }
                 return;
+            } catch (StaleElementReferenceException ex) {
+                // item went away while we were trying to click (like auto complete)
+                // so try again
```

Comment:
Why do I feel we're swimming upstream here?

Reply:
I mean you are not wrong. But in this case, javascript destroys the element we just got before we had a chance to do something with it. I think thats a safe thing to catch most places. We'll see